### ISSUE TITLE
Getting error with main code inside beatifulSoup

### ISSUE THREAD

**Issue by tienquyet28** [taspinar/twitterscraper#372]
[/usr/local/lib/python3.7/dist-packages/twitterscraper/query.py](https://localhost:8080/#) in get_proxies()
     47     soup = BeautifulSoup(response.text, 'lxml')
     48     table = soup.find('table',id='proxylisttable')
---> 49     list_tr = table.find_all('tr')
     50     list_td = [elem.find_all('td') for elem in list_tr]
     51     list_td = list(filter(None, list_td))

AttributeError: 'NoneType' object has no attribute 'find_all'

When I run CLI or in python It get error from BeautifulSoup lib, please help

**Comment by tsp2123** [comment]
Has anyone found a solve for this?

**Comment by Prit2341** [comment]
hey I know how to solve this issue I would Like to work on this

**Comment by Prit2341** [comment]
D:\twitterscraper\twitterscraper>python main.py

D:\twitterscraper\twitterscraper>

**I am getting this output is this right or wrong**

**Comment by jkahil** [comment]
yes just put soup.find('table') as the table id has been dropped from the page

**Comment by CryptoRahino** [comment]
got hte same issue.
solved this with 
removing `,id='proxylisttable'` from `table = soup.find('table',id='proxylisttable')`
in query.py > `get_proxies()`
